lib/src/functions/math.dart

```diff
@@ -21,14 +22,16 @@ final global = UnmodifiableListView([
   _round, _ceil, _floor, _abs, _max, _min, _randomFunction, _unit, //
   _percentage,
   _isUnitless.withName("unitless"),
-  _compatible.withName("comparable")
+  _compatible.withName("comparable"),
+  _div.withName("divide")
 ]);
 
 /// The Sass math module.
 final module = BuiltInModule("math", functions: [
   _round, _ceil, _floor, _abs, _max, _min, _randomFunction, _unit,
   _isUnitless, //
-  _percentage, _compatible
+  _percentage, _compatible,
+  _clamp, _div, _hypot
```

Comment:
I don't know why the formatting got weird here before, but this should be flowed onto as few lines as possible:

```dart
final module = BuiltInModule("math", functions: [
  _round, _ceil, _floor, _abs, _max, _min, _randomFunction, _unit, //
  _isUnitless, _percentage, _compatible, _clamp, _div, _hypot
]);
```

Reply:
Done. I also decided to reorganize this file, let me know your thoughts.

Reply:
LGTM